modules/mono/build_scripts/build_assemblies.py

```diff
@@ -342,6 +342,13 @@ def build_all(msbuild_tool, module_dir, output_dir, godot_platform, dev_debug, p
 def main():
     import argparse
     import sys
+    import os.path
+
+    if not os.path.exists("modules/mono/glue/GodotSharp/GodotSharp/Generated/"):
```

Comment:
Probably a bit safer to check for `GodotSharp/Generated/GeneratedIncludes.props`.